backend/src/functions/ingestContract/contract.ts

```diff
@@ -0,0 +1,11 @@
+interface Contract {
```

Comment:
To use this interface in other files you have to `export` it. 

`export interface Contract ...`